### PR TITLE
Gradle 6.2.

### PR DESCRIPTION
**Pull request by ychescale9**
Let's see if this still breaks codecov.

### CODE REVIEW

**Comment by digitalbuddha** [comment]
Weird! Test failure that succeeded on ci first time. I guess I wrote a flaky test

File: store-rx2/src/test/kotlin/com/dropbox/store/rx2/test/RxSingleStoreTest.kt [hunk 1]
```diff
@@ -50,7 +50,7 @@ class RxSingleStoreTest {
                     Completable.complete()
                 }
             )
-            .withScheduler(Schedulers.io())
+            .withScheduler(Schedulers.single())
```
Comment (pavlospt) [inline store-rx2/src/test/kotlin/com/dropbox/store/rx2/test/RxSingleStoreTest.kt]: Since this is a test case, you can also pass `Schedulers.trampoline()`.

**Comment by ychescale9** [comment]
I changed the `scheduler` to `Schedulers.single()`. Let's see if that helps.

Also it looks like Gradle didn't cause the codecov issue after all...

File: store-rx2/src/test/kotlin/com/dropbox/store/rx2/test/RxSingleStoreTest.kt (hunk 1 above)
Comment (ychescale9) [reply store-rx2/src/test/kotlin/com/dropbox/store/rx2/test/RxSingleStoreTest.kt]: Now this is getting more interesting. After changing it to `trampoline()`, the test fails consistently:

```
java.lang.AssertionError: Values at position 1 differ; expected: Data(value=3 2, origin=Fetcher) (class: Data) but was: Data(value=3 2, origin=Persister) (class: Data) (latch = 1, values = 2, errors = 0, completions = 0)
Expected :Data(value=3 2, origin=Fetcher) (class: Data) 
Actual   :Data(value=3 2, origin=Persister) (class: Data) (latch = 1, values = 2, errors = 0, completions = 0)
```

Any idea?

**Comment by ychescale9** [comment]
Ok that didn't help, reverted as this is a separate issue.

**Comment by pavlospt** [comment]
@ychescale9 I can take a look at it! :D

**Comment by ychescale9** [comment]
And now it passed.. and codecov is back. Definitely a flaky test...

**Comment by ychescale9** [comment]
@pavlospt Thanks! Maybe create a separate PR as it's unrelated to this?

**Comment by pavlospt** [comment]
@ychescale9 yeah totally, will create a new one!

**Pull request merged**